### ISSUE TITLE
Custom OS cursor breaking for macOS since recent update?

### ISSUE THREAD

**Issue by subsoap** [subsoap/defos#110]
>In FSH since last update the mouse cursor is now the system cursor and switches to game cursor when over something clickable. It also sometimes flashes between the 2 when over a clickable card.

10.9.5

**Comment by subsoap** [comment]
Not sure if it's our  fault but we have not touched that code in forever in our game.

**Comment by britzl** [comment]
@jcash might have some input here. Did we make any changes to the macOS application startup process or was this only for iOS? Also, the merge of the Vulkan stuff, could this have some impact on mouse cursors? Or did we update the build pipeline on macOS maybe?

**Comment by JCash** [comment]
No, there are no changes to the OSX app that should have caused this imo.
We have done some vulkan preparation (it's not enabled yet) so we did some preparation for it, but I don't actually suspect that part.

Can you pinpoint at what engine version this started to happen?

**Comment by dapetcu21** [comment]
I don't have the same issue in Interrogation. The custom cursor works fine.

**Comment by subsoap** [comment]
Engine version is 1.2.164. Unfortunately we have very few macOS players so I only have info from one person. I'll try to get more people to test and see if they experience the same issue.

**Comment by dapetcu21** [comment]
@subsoap Is this FSH? I can test that if you want to.

**Comment by subsoap** [comment]
Yes, it is. Latest should be the same, make sure custom hardware cursor is active in settings then try to hover buttons and such. See if system cursor appears randomly while doing so.

**Comment by dapetcu21** [comment]
@subsoap check slack pls

**Comment by britzl** [comment]
It works for me with 1.1.19.11.25 on Mojave

**Comment by britzl** [comment]
Although if you move the mouse cursor back and forth very quickly the accessibility thing that shows a magnified mouse cursor appears instead of the custom one.

**Comment by dapetcu21** [comment]
Yeah, no issues here with 1.1.19.11.25 on Catalina

**Comment by subsoap** [comment]
Need to make a tester app which relatively quickly swaps the custom cursor then to see if any system cursor appears when it should not.
